node/express/getting-started/app.ts

```diff
@@ -10,14 +10,14 @@ interface Product {
     brand: string;
   };
 
-const getProducts = (request: Request, response: Response, next: NextFunction) => {
+const getProducts = (request: Request, libResponse: Response, next: NextFunction) => {
```

Comment:
Same change as above and below. You probably did some refactoring. Please remove all changes outside of the retrofit folder from this PR.